packages/ensnode-sdk/src/ensanalytics/types.ts

```diff
@@ -0,0 +1,184 @@
+import type { Address } from "viem";
+
+import type { Duration, UnixTimestamp } from "../shared";
+
+/**
+ * The default number of items per page for paginated aggregated referrer queries.
+ */
+export const PAGINATION_DEFAULT_LIMIT = 25;
```

Comment:
```suggestion
export const ITEMS_PER_PAGE_DEFAULT = 25;
```

Goal: Align with the renaming of `PaginatedAggregatedReferrersRequest.itemsPerPage`.